helpers.md

```diff
@@ -844,6 +845,25 @@ The `Arr::reject` method removes items from an array using the given closure:
 
     // [0 => 100, 2 => 300, 4 => 500]
 
+<a name="method-array-select"></a>
+#### `Arr::select()` {.collection-method}
+
+The `Arr::select` method select an array of values from an array:
+
+```php
+use Illuminate\Support\Arr;
```

Comment:
According to #10186, we prefer having PHP opening tags in the code sample blocks:
```suggestion
<?php

use Illuminate\Support\Arr;
```

Reply:
Thanks for your suggestion. Added.